### ISSUE TITLE
[ANNOUNCEMENT] Hyperlearn revamp mid 2022!

### ISSUE THREAD

**Issue by danielhanchen** [danielhanchen/hyperlearn#21]
### Hello Hyperlearn fans!
_At Moonshot, our mission is to predict the future of everything quickly and make JARVIS a reality!_

I was on hiatus in late 2018 to early 2020 since I was at NVIDIA helping make GPU ML algos faster!
I incorporated Hyperlearn's methods to make TSNE 2000x faster and various other ML algos faster!
From 2020 to now, I've been working on Moonshot with my brother, so I didn't have too much time to maintain Hyperlearn!
But things been catching up so a polished version will be published mid next year!! 

_Did you know Hyperlearn's methods has been used / mentioned in 5 research papers from Microsoft, UW, Greece, UC Berkeley and NVIDIA? Also Hyperlearn's methods and effort were incorporated into Cupy, Pytorch, Scipy, UNSW and GCC?_

**Around mid 2022, Hyperlearn will evolve to GreenAI and aims to incorporate:**

- [ ] New Paratrooper optimizer - fastest SGD variant combining Lookahead, Learning Rate Range Finder, and more!
- [ ] 30% faster Matrix Multiplication on CPUs
- [ ] Software Support for brain floating point (bfloat16) on nearly all hardware
- [x]  Easy compilation on old and new CPU hardware (x86, ARM)
- [ ] 100x faster regular expressions
- [ ] 50% faster and 50% less memory usage for assembly kernel accelerated methods
- [x] Fast and parallelized New York Times scraper
- [x] Fast and parallelized NYSE Announcements scraper
- [x] Fast and parallelized FRED scraper
- [x] Fast and parallelized Yahoo Finance scraper
